modules/social.py

```diff
@@ -42,7 +42,9 @@ def __init__(self, bot):
             "profile": re.compile(r'(?:sw)?[ \-\u2014]?(\d{4})[ \-\u2014]?(\d{4})[ \-\u2014]?(\d{4})', re.I),
             # Chat filter, "It appears you've sent a friend code." Requires separators and discards select prefixes.
             # Discarded prefixes: MA/MO (AC Designer), DA (AC Dream Address).
-            "chatFilter": re.compile(r'(sw|m[^ao]|d[^a]|[^MD]\w|^\w|^)[ \-\u2014]?\d{4}[ \-\u2014]\d{4}[ \-\u2014]\d{4}', re.I + re.M)
+            "chatFilter": re.compile(r'(sw|m[^ao]|d[^a]|[^MD]\w|^\w|^)[ \-\u2014]?\d{4}[ \-\u2014]\d{4}[ \-\u2014]\d{4}', re.I + re.M),
+            # super mario maker friend code
+            "smm": re.compile(r'(\d{3})[ \-\u2014]?(\d{3})[ \-\u2014]?(\d{3})', re.IGNORECASE)
```

Comment:
* `friendCodeRegex` implies being a friend code, not sure this should be in this object.
* SMM2 Marker Codes can contain alphanumeric characters so this should be `[\da-z]` not `\d`
* Other complied regex use `re.I` shorthand instead of `re.IGNORECASE`